lib/facilities/drivetime_bands/response.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'common/models/base'
+
+module Facilities
+  module DrivetimeBands
+    class Response < Common::Base
```

Comment:
What is the purpose of this class? I don't see where it's used

Reply:
Ah, looks like a change of mine got squashed 😞I ran into a big merge that I had to go through manually. Thanks for this catch 🙇‍♀